### ISSUE TITLE
Self interaction not working

### ISSUE THREAD

**Issue by TheAngryPacifist** [acemod/ACE3#5328]
**Arma 3 Version:** `1.72.142223` (stable)
**CBA Version:** `3.4.0.170627` (stable)
**ACE3 Version:** `3.10.1` (stable)

**Mods:**
```
- CBA_A3
- ace
-Overthrow
```

**Description:**
- Self interaction menu not appearing. Only a red circle shows up. 

**Steps to reproduce:**
- Start up a the game.
- Go into a game
- Try to use self interaction 

**Where did the issue occur?**
- Self-Hosted Multiplayer / Singleplayer

**Placed Modules:**
- Overthrow

**Comment by Nicholaiii** [comment]
Exactly the same issue here. Tried with CBA 3.3.1 and 3.4.0

**Comment by bux** [comment]
Ref: https://github.com/acemod/ACE3/issues/5323

**Comment by bux** [comment]
@TheAngryPacifist @Nicholaiii 
You both forgot supplying an RPT file as requested in the issue template.

**Comment by Tralin** [comment]
Sorry, didn't realize the other issue was closed.

Copy+paste from the other issue:

I'm experiencing this problem. Using only ACE and CBA (newest versions, downloaded them from GitHub last night) I cannot use the self-interaction menu at all. It's just a red circle.

Tried by using a Workshop missions, single player scenarios, and even made a new mission in the Editor.

Tried creating a new profile, same result. Tried both Workshop ACE and CBA, and GitHub versions, same result. Tried resetting all parameters in Arma 3 launcher, same result.

RPT file here: https://pastebin.com/87uaQMcD

An observation:

Myself, maximemoring in #5323, and TheAngryPacifist are all using Overthrow. I've tried in-and-out of Overthrow, and I disabled Overthrow while testing. But it is similar to each of us.

I should mention that a friend of mine playing on my Overthrow server has no issues with it at all. He also has Overthrow installed.
